hooks/pre/30-upgrade.rb

```diff
@@ -80,6 +80,22 @@ def fix_katello_settings_file
   end
 end
 
+def mongo_mmapv1_check
+  custom_hiera = '/etc/foreman-installer/custom-hiera.yaml'
+  # check and see if we have a pulp_database already and if the Hiera file has already been modifed.
+  if File.file?('/var/lib/mongodb/pulp_database.0')
```

Comment:
Is this file always an indication of `mmapv1` or is it also present with `wiredTiger`?

Reply:
@ekohl it would be present with `wiredTiger` too

Reply:
Is there a way to detect the database format? If this hook is running on a system that for some reason is already wiredTiger (database restore?) it'll break. It'd be great if we could detect that.